server/remote/bitbucketserver/internal/client.go

```diff
@@ -233,9 +241,22 @@ func (c *Client) GetHooks(owner string, name string) (*HookSettings, error) {
 
 //TODO: make these as as general do with the action
 
+//Helper function to help create get
+func (c *Client) doGet(url string) (*http.Response, error) {
+	request, err := http.NewRequestWithContext(c.ctx, "PUT", url, nil)
```

Comment:
```suggestion
	request, err := http.NewRequestWithContext(c.ctx, "GET", url, nil)
```

Reply:
Good catch missed that ... (copy/paste issue)